pandas/tools/tests/test_pivot.py

```diff
@@ -936,6 +936,84 @@ def test_crosstab_no_overlap(self):
 
         tm.assert_frame_equal(actual, expected)
 
+    def test_margin_ignore_dropna_bug(self):
+        # GH 12577
```

Comment:
yes that sounds fine (and just add this number after the other one). Idea is to have same tests together even though they occurred via different issues.